app/views/miq_policy/show.html.haml

```diff
@@ -1,7 +1,7 @@
 #policy_details_div
   -  policy_conditions = @record.conditions
   -  policy_events = @record.miq_event_definitions
-  -  expression_table = @record.expression.kind_of?(MiqExpression) ? exp_build_table(@record.expression.exp) : nil
+  -  expression_table = @record.expression.kind_of?(MiqExpression) ? build_expression_table : nil
```

Comment:
why not just put the `nil` logic inside `build_expression_table` method?

Reply:
done